packages/lib/gpcircuits/scripts/copy-snark-artifacts.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 GPCIRCUITS_PACKAGE_DIR="$SCRIPT_DIR/.."
-SRC_DIR="$GPCIRCUITS_PACKAGE_DIR/artifacts/test"
+SRC_DIR="$GPCIRCUITS_PACKAGE_DIR/artifacts/prod"
```

Comment:
Make sure to update the docs in the snark-artifacts repo to specify how to generate and test a new set of prod artifacts before releasing to NPM.
This commented isn't limited to this file, or even this PR, but this file is where I thought of it.